sunpy/net/dataretriever/sources/rhessi.py

```diff
@@ -50,14 +50,13 @@ class RHESSIClient(GenericClient):
 
     >>> from sunpy.net import Fido, attrs as a
     >>> results = Fido.search(a.Time("2016/1/1", "2016/1/2"),
-    ...                       a.Instrument('RHESSI'))  #doctest: +REMOTE_DATA
+    ...                       a.Instrument("RHESSI"), a.Physobs("summary_lightcurve"))  #doctest: +REMOTE_DATA
```

Comment:
This count as a breaking change?

Reply:
yeah, doing `a.Time(), a.Instrument("RHESSI")` now returns results from both our `RHESSIClient` and `VSOClient` so it's giving you *more* than before. (If you *just* want `RHESSIClient` results you have to add this).

Reply:
More changelogs!